cves/kernel/CVE-2017-0861.yml

```diff
@@ -388,38 +395,38 @@ lessons:
 
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
-  defense_in_depth:
-    applies:
+  defense_in_depth: 
+    applies: False
     note:
   least_privilege:
-    applies:
+    applies: False
```

Comment:
Based on the description of the vulnerability provided by NVD, it sounds like this lesson, along with the "distrust input" lesson could apply to this vulnerability from the part that says "allows attackers to gain privileges via unspecified vectors." Based on your research of this vulnerability, would you agree or disagree?

Reply:
Disagree.  In one of the linux logs regarding this issue, it is said that elevation of privilege cannot be completely ruled out, but is very unlikely.